#exception -- IndexError: list index out of range

my_list=[2,9,8]

print(my_list[5])